src/animation/AnimationAction.js

```diff
@@ -13,6 +13,10 @@ import { WrapAroundEnding, ZeroCurvatureEnding, ZeroSlopeEnding, LoopPingPong, L
 
 function AnimationAction( mixer, clip, localRoot ) {
 
+	if ( ! mixer ) throw new Error( "Mixer can't be null or undefined !" );
```

Comment:
Not sure how I feel about these about these...

I guess it's time for me to try that part of the API 😇

Reply:
I just adjusted the code to meet the test expectations. If the code will not be throwing, the test should not expect the exceptions. Adjusting the test instead is just as easy, just say which way you prefer it.

Reply:
I think I prefer adjusting the tests instead.